aiohttp/web.py

```diff
@@ -422,6 +422,10 @@ def run_app(app, *, host=None, port=None, path=None, sock=None,
     app._set_loop(loop)
     loop.run_until_complete(app.startup())
 
+    if print is None:
```

Comment:
much easier just do `if print` below before calling it.

Reply:
I guess, if that's the only place you ever plan to use that print argument?

Reply:
There, I updated it.

Reply:
looks like you having a linting error :smile:

Reply:
Woops, failed my block selection. There you go, sorry :-P